MHdrift

pedir opened the pull request:
This adds drifting proposals to MH. Also adds burn-in, graphing of the trace, gweke test, and refactors SMC to use MH methods when possible.

pedir commented:
The travis-ci failure seems to be just an npm warning not related to these changes. The tests and lints are all passing.

stuhlmueller commented:
Agree that the travis failure looks unrelated (would be good to fix it, though).

Can you add the equivalent of testDriftLinearRegression.wppl but make it use SMC? (Just to verify that drift proposals also work as part of SMC.)

pedir commented:
I fixed the npm warning on dev so that shouldn't be an issue anymore.

I also added a test for drift PFRj very similar to the linear regression one for the MHDrift. They will hopefully be consolidated when we get around issue #94.
